HHS/Head-Start-TTADP
# 12/02 Fix audit vulns

## Description of change

12/02 Fix audit vulns:

There are three audit vulns 2 we can use the patched version (**minimatch** and **decode-uri-component**). The third **loader-utils** doesn't support the patched version. However because its used by react-scripts can be safely ignored.

## How to test

Run yarn deps. The site should start up fine and be usable. All tests should pass.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
